internal/core/src/storage/prometheus_client.cpp

```diff
@@ -148,6 +148,22 @@ DEFINE_PROMETHEUS_COUNTER(internal_storage_op_count_remove_fail,
                           internal_storage_op_count,
                           removeFailMap)
 
+//load metrics
+std::map<std::string, std::string> downloadDurationLabels{{"type", "download"}};
+std::map<std::string, std::string> writeDiskDurationLabels{{"type", "write_disk"}};
+std::map<std::string, std::string> deserializeDurationLabels{{"type", "deserialize"}};
+DEFINE_PROMETHEUS_HISTOGRAM_FAMILY(internal_storage_load_duration,
+                                   "[cpp]durations of load segment")
+DEFINE_PROMETHEUS_HISTOGRAM(internal_storage_download_duration,
+                            internal_storage_load_duration,
+                            downloadDurationLabels)
+DEFINE_PROMETHEUS_HISTOGRAM(internal_storage_write_disk_duration,
+                            internal_storage_load_duration,
+                            writeDiskDurationLabels)
+DEFINE_PROMETHEUS_HISTOGRAM(internal_storage_deserialize_duration,
```

Comment:
default minimum bucket is 1ms, we may not see any distribution of deserialize duration?